frame/beefy/src/lib.rs

```diff
@@ -120,17 +127,23 @@ impl<T: Config> Pallet<T> {
 		<NextAuthorities<T>>::put(&queued);
 	}
 
-	fn initialize_authorities(authorities: &[T::BeefyId]) {
+	fn initialize_authorities(authorities: &[T::BeefyId]) -> Result<(), ()> {
 		if authorities.is_empty() {
-			return
+			return Ok(())
 		}
 
-		assert!(<Authorities<T>>::get().is_empty(), "Authorities are already initialized!");
+		if !<Authorities<T>>::get().is_empty() {
+			return Err(())
+		}
 
-		<Authorities<T>>::put(authorities);
+		let bounded_authorities =
+			BoundedSlice::<T::BeefyId, T::MaxAuthorities>::try_from(authorities)?;
+
+		<Authorities<T>>::put(bounded_authorities);
```

Comment:
`BoundedSlice` is not `Copy`:
```suggestion
		<Authorities<T>>::put(bounded_authorities.clone());
```

Reply:
It IS `Copy`, my code has updated to make it so, I don't know why CI isn't picking it up.

Reply:
This is absolutely nuts, running `cargo check -p pallet-beefy` is fine locally, but CI is not picking up my changes to `BoundedSlice`.

Reply:
I think there's a caching problem in CI where it does not correctly identify _new dirty_ files it should pick up...
Merging master on top of this PR might be a workaround here.